web/src/types/index.ts

```diff
@@ -345,3 +345,22 @@ export type SnapshotSettings = {
     hostPath: { path: string };
   };
 };
+
+export type ConfigGroupItem = {
+  name: string;
+  title: string;
+  type: string;
+  hidden: boolean;
+  validationError: string;
+  error: string;
```

Comment:
Changed these 2 values to string VS boolean based on the types we had defined in `AppConfig` and what I could find in the code (I do believe these should be strings). TSC said ok so I believe we're fine.